.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+ignore = W503
+exclude = 
+    snoop/data/migrations/,
```

Comment:
Instead of excluding the migrations, could we put this under `per-test-ignores`? (just need to ignore max line length I think)

Once every half a year we end up writing migrations by hand and it would be really useful to have this check here.